File: CQRS-write/cqrswrite-infrastructure/src/main/java/com/example/cqrswrite/domain/feed/event/vo/CreateFeedEvent.java
package com.example.cqrswrite.domain.feed.event.vo;

import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;
import java.util.UUID;

@Getter
@NoArgsConstructor
public class CreateFeedEvent {

    private UUID id;
    private String title;
    private String content;
    private String username;
    private LocalDateTime createdAt;

    @Builder
    public CreateFeedEvent(UUID id, String title, String content, String username, LocalDateTime createdAt) {
        this.id = id;
        this.title = title;
        this.content = content;
        this.username = username;
        this.createdAt = createdAt;
    }
}

File: CQRS-read/cqrsread-infrastructure/src/main/kotlin/com/example/cqrsread/domain/feed/event/vo/CreateFeedEvent.kt
package com.example.cqrsread.domain.feed.event.vo

import java.time.LocalDateTime
import java.util.UUID

data class CreateFeedEvent(
    val id: UUID,
    val title: String,
    val content: String,
    val username: String,
    val createdAt: LocalDateTime
)
